frontend/public/locales/fr/translation.json

```diff
@@ -821,7 +821,7 @@
   "Copy button": "Bouton Copier",
   "Copy this private URL to share": "Copier cette URL privée à partager",
   "Copy to clipboard": "Copier dans le Presse-papiers",
-  "copy.login.command.instructions": "Utilisez masterhead pour accéder à l’option <bold>de commande Copier la connexion</bold> sous votre profil utilisateur. Exécutez la commande <code>oc login</code> fournie pour vous connecter.",
+  "copy.login.command.instructions": "Utilisez masterhead pour accéder à l’option <bold>Copier la commande de connexion</bold> sous votre profil utilisateur. Exécutez la commande <code>oc login</code> fournie pour vous connecter.",
```

Comment:
The text inside the `<bold>` tags refers to a menu item in the UI belonging to OpenShift console. When using our product in French, the OpenShift console will also be in French, so this string needs to match exactly the translation that is used in OpenShift.